### DISCUSSION TITLE
Update gr-satellites under Conda environment

### DISCUSSION THREAD

**Discussion by Vinz87** [daniestevez/gr-satellites#528] [Q&A]
Hi, sometime ago I successfully followed the indications in the [documentation](https://gr-satellites.readthedocs.io/en/latest/installation_conda.html) to install GNURadio and gr-satellites under the Conda virtual environment, on a Macbook Pro M1.

gr-satellites says it is at v5.2.0, what should I do in the Conda environment to update gr-satellites (and possibly GNURadio too) to the latest available version?

Thanks

**Comment by daniestevez** [comment]
https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-pkgs.html#updating-packages

**Reply by Vinz87** [reply]
Hi, I tried that, but it seems that whatever I do conda doesn't update packages.
I first tried with the `update` command, both for `gnuradio-companion` and `gnuradio-satellites` packages, and both of them will simply remain at the same version after the command execution.
I also tried removing the `gnuradio-satellites` package, and then reinstalling it, and it will reinstall my current version v5.2.0 instead of the last available one v5.5.0 (which shows up with the `conda search` command)
